SKETCH.md

```diff
@@ -0,0 +1,1440 @@
+
+Type: free form (optional)
+
+> **justification**
+> 
+> More detail on the type of situations where people have felt unwelcome can let us better 
+> address these issues in the future.
+
+
+### Please let us know your feeling about the following PureScript community working groups.
+
+Type: matrix (optional)
+
+Group:
+
+- Compiler
+- Community governance
+- Contrib
+- Node
+- Web
+- Registry
+- Survey
+
+Sentiment:
+
+- Please _don't_ work on this, it's a distraction.
+- If someone wants to work on it, sure, but I don't really have an opinion.
+- I hesitate saying anything for fear of it being interpreted as an opinion, which I fully lack.
+- I'd dig it if the working group produced useful stuff, but I could live without it too.
+- I would love to see this worked on.
+- This is super important. I am working on this, will work on this myself, or pay a bounty for this to be solved.
+
+## PureScript at work
+
+### To what extent is PureScript currently being used by your company?
+
+Type: select one
+
+- My company uses PureScript for a large portion of production projects
+- My company uses PureScript for a small portion of production projects
+- My company uses PureScript only for non-production projects (e.g., tooling)
+- My company has actively experimented with PureScript
+- My company has seriously considered, but not experimented with, using PureScript
+- My company has not seriously considered PureScript for any use
+- I am unsure whether my company has considered using or currently uses PureScript
+- I don't work for a company or my company does not develop software of any kind [`NEXT`](#purescript-in-education)
+
+> **justification**
+>
+> We want to establish how reliant companies are on PureScript.
+
+### Approximately how many total developers does your company employ?
+
+**Note**: don't worry about being exact here! Go with you gut.
+
+Type: select one (optional)
+
+- Under 10
+- 11-49
+- 50-99
+- 100-500
+- 500-1,000
+- 1,000-10,000
+- Over 10,000
+
+> This question is not that interesting on its own, but it can be used as a sort of co-hort for understanding how answers 
+> change depending on the size of the development effort at a company.
+>
+> Previously this question used "employees" instead of "developers". It is more appropriate for us to ask about the amount
+> of developers at a company vs. the amount of people employed in total.
+
+### Is your company planning on hiring PureScript developers in the next year?
+
+Type: select one (optional)
+
+- Yes
+- No
+- I don't know
+
+> This question assess hiring sentiment. Although there is intrinsic uncertainty, it is easy to answer and forward looking.
+> It will also be interesting to see what the demand for PureScript skills from companies is over time.
+
+### In what technology domain(s) is PureScript used at your company?
```

Comment:
Since PureScript's main backend is JavaScript, shouldn't there be a question like "If using PureScript for web dev, which industry does your web app target?" This question feels very heavily influenced by Rust. For example, we list the "HPC (High-performance [Super]Computing)". That makes sense for Rust, but does it make sense for us here?

Reply:
I'm not sure how much folks are using the C++, Go, and C backends, and if they are then theoretically anything in Rust-land is fair game here. However, I agree that webdev deserves its own question, and the one you propose sounds great! I'll add it.